vizalerts.py

```diff
@@ -1045,29 +1044,41 @@ def get_password_from_file(password):
         sys.exit(1)
 
 
-def find_vizimages_in_body(data, has_email_header, has_email_footer):
-    """Returns list of all found occurences of VIZ_IMAGE() in email body text"""
+
+def find_vizimages_in_body(data, viewurlsuffix, has_email_header, has_email_footer):
+    # updated 20160223 Jonathan Drummey
```

Comment:
I’m happy to change that code... I’ve been wondering about that - where I’ve made “big" changes to the code how would you like me to comment them?
--
Jonathan Drummey - 207.831.9657 - jonathan.drummey@gmail.com
http://drawingwithnumbers.artisart.org <http://drawingwithnumbers.artisart.org/>



> On Mar 1, 2016, at 4:27 PM, Matt <notifications@github.com> wrote:
> 
> In vizalerts.py <https://github.com/tableau/VizAlerts/pull/14#discussion_r54638434>:
> 
> > @@ -1045,29 +1044,41 @@ def get_password_from_file(password):
> >          sys.exit(1)
> >  
> >  
> > -def find_vizimages_in_body(data, has_email_header, has_email_footer):
> > -    """Returns list of all found occurences of VIZ_IMAGE() in email body text"""
> > +
> > +def find_vizimages_in_body(data, viewurlsuffix, has_email_header, has_email_footer):
> > +    # updated 20160223 Jonathan Drummey
> Minor quibble, but would you mind changing the comment block to this:
> 
> def find_vizimages_in_body(data, viewurlsuffix, has_email_header, has_email_footer):
> """
> Return a list of all viewnames to be downloaded for the
> body, header, and/or footer. The list is a distinct list
> to prevent unnecessary downloading.
> updated 20160223 Jonathan Drummey
> function call now includes viewurlsuffix, return is now a list of viewnames
> """
> 
> That way the docstring can be used in documentation (should we ever need it)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tableau/VizAlerts/pull/14/files#r54638434>.
>